spec/features/user_can_delete_their_wanted_and_had_skills_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.